tests/unit/utils/test_logger.py

```diff
@@ -16,6 +18,15 @@ def remove_logfile():
         log_file_name.unlink()
 
 
+def unset_log_env_vars():
+    """Unset environment variables about logging."""
+    os.environ.pop("LOG_LEVEL", None)
+    os.environ.pop("LOG_LEVEL_CONSOLE", None)
+    os.environ.pop("LOG_LEVEL_FILE", None)
+    os.environ.pop("LOG_FILE_NAME", None)
+    os.environ.pop("LOG_FILE_SIZE", None)
```

Comment:
This https://github.com/openshift/lightspeed-service/pull/152 removes a bunch of envs related to logging to a file. Just `LOG_LEVEL` would be left. With that, consider using
```python
@patch.dict("os.environ", {"SOME_ENV": "value"})
def test_blabla...
```
as you would be able to set the envs exactly according to particular test needs.